src/components/advanced-marker.tsx

```diff
@@ -31,6 +31,10 @@ export function isAdvancedMarker(
   );
 }
 
+function isElementNode(node: Node): node is HTMLElement {
+  return (node as Node).nodeType === Node.ELEMENT_NODE;
```

Comment:
isn't `as Node` redundant here, since that's the type given above?

Reply:
Ah yes, you're right.